test/sparse/sparse.jl

```diff
@@ -1779,6 +1779,21 @@ end
     show(ioc, MIME"text/plain"(), sparse(Int64[1, 1], Int64[1, 2], [1.0, 2.0]))
     @test String(take!(io)) == "1×2 SparseMatrixCSC{Float64,Int64} with 2 stored entries:\n  ⋮"
 
+    # even number of rows
+    ioc = IOContext(IOContext(io, :displaysize => (8, 80)), :limit => true)
```

Comment:
Could simplify the nested `IOContext` calls to `ioc = IOContext(io, displaysize = (8, 80), limit = true)` (assuming I understand the docstring correctly and my experiments at the REPL confirm what they appear to)?